addon/components/s-step.js

```diff
@@ -14,17 +14,17 @@ export default Component.extend({
   }),
   iconName: computed('isActive', function () {
     if(this.get('isActive') === 'active') {
-      return 'edit-2';  
+      return 'edit-2';
     }
     else {
       return 'check';
     }
-  }),  
+  }),
   actions: {
     callback() {
-      if((this.get('key') < this.get('current')) && this.get('onClick')) {
+      if((this.get('key') !== this.get('current')) && this.get('onClick')) {
```

Comment:
why this is changed?

Reply:
Because there should be a way to move backwards also.

Reply:
This was done intentionally. Most of the UI components work in the same way.  If we allow this, it will make inconsistency across steps for completed steps. A function is provided to go back step